devel/yaml-cpp/Portfile

```diff
@@ -39,3 +39,14 @@ configure.args      -DYAML_BUILD_SHARED_LIBS=ON
 # https://github.com/jbeder/yaml-cpp/issues/539
 configure.args-append \
                     -DYAML_CPP_BUILD_TESTS=OFF
+
+variant tools description {Install parse tools with suffix yaml-*} {
+    post-destroot {
+        xinstall -m 0755 ${workpath}/build/util/parse \
+            ${destroot}${prefix}/bin/yaml-parse
+        xinstall -m 0755 ${workpath}/build/util/read \
+            ${destroot}${prefix}/bin/yaml-read
+        xinstall -m 0755 ${workpath}/build/util/sandbox \
+            ${destroot}${prefix}/bin/yaml-sandbox
+    }
+}
```

Comment:
`read` conflicts with the `read` utility from `coreutils`. Therefore, in this way I decided to avoid conflict. The popular distributions I've looked at don't use these tools.

Reply:
ah, okay. So it seems that these binaries are not installed by default - correct? So if you think they are useful (but need the rename) then you could just do that without the variant I'd say. Perhaps you just add a `note` saying that they are installed with the `yaml-` suffix or something like that.

Reply:
`parse` or `read` has like the `yamllint` indicate syntax errors, at least for obvious mistakes.

As for me, the project most uses as a library. But I think one day someone will need these tools and the user will install them with this variant. I added notes to the variant.